public/the_bridge_templates/marketplace/index.html

```diff
@@ -59,11 +59,21 @@ <h3 class="title">
                 DETAILS
               </a>
               -->
-              <a ng-href="#" class="details-link">
+              <a ng-click="changeVote('up', product.id)" class="details-link up_{{product.id}}" style="margin-right:40px;">
```

Comment:
Please don't use dynamic class names. You have access to `product` from the `ng-repeat`.